docs/content/docs/start/quick-start.md

```diff
@@ -14,7 +14,7 @@ localhost.
 
 {{< callout note >}}
 
-The demo app uses Fluid Framework 2.0, which is in preview.
+The demo app uses Fluid Framework 2.
```

Comment:
2.0 feels better than 2, I think.
```suggestion
The demo app uses Fluid Framework 2.0.
```

Reply:
Going with 2.X for consistency and future proofing.